Examples/restcomm-olympus-nopod/restcomm-olympus/Utils.m

```diff
@@ -565,5 +643,25 @@ + (void)shakeView:(UIView *)view {
     [view.layer addAnimation:shake forKey:@"position"];
 }
 
++ (void)shakeTableViewCell:(UITableViewCell *)cell{
+    CGPoint position = cell.center;
+    
+    UIBezierPath *path = [UIBezierPath bezierPath];
+    [path moveToPoint:CGPointMake(position.x, position.y)];
```

Comment:
I was ready to ask you to remove it and make it consistent with other settings screens where we get the dull alert, for consistency, but couldn't see this effect go away :D. Anyway, let's leave it like that for now; we can discuss in terms of consistency when we revisit the UI/UX of Olympus as a whole

Reply:
Sure.